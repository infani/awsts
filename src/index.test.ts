import * as awsts from './index';

describe('index', () => {
    it('index', async () => {
        expect(awsts).toMatchSnapshot();
    });
});