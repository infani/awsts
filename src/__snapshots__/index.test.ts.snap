// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index index 1`] = `
Object {
  "AppSync": Object {
    "GetApiKey": [Function],
  },
  "CloudMap": Object {
    "DiscoverInstances": [Function],
  },
  "Cognito": Object {
    "Configure": [Function],
    "GetJwtToken": [Function],
  },
  "IotData": Object {
    "Publish": [Function],
  },
  "Opensearch": Object {
    "NewClient": [Function],
  },
  "SSM": Object {
    "GetParameter": [Function],
  },
}
`;
